### ISSUE TITLE
Scrolling is disabled when listening to swipe-Events

### ISSUE THREAD

**Issue by newnottakenname** [driftyco/ionic#6731]
#### Short description of the problem:

When listening inside of an ion-list for a swipe event, the list can no longer scroll

```
<ion-list>
   <ion-item (swipe)="swipeEvent(event)"> </ion-item>
</ion-list>
```

View the [plunker demo here](http://plnkr.co/edit/GSQWf2FzHCf69mZUrEo0)

**Ionic Version:** 2.0.0-beta.7

Ionic Framework Version: 2.0.0-beta.7
Ionic CLI Version: 2.0.0-beta.25
Ionic App Lib Version: 2.0.0-beta.15
OS: Distributor ID:	elementary OS Description:	elementary OS Freya 
Node Version: v4.4.5

**Comment by jgw96** [comment]
Hello! Thanks for opening an issue with us! Would you mind posting a plunker i can use to reproduce this issue? Thanks!

**Comment by manucorporat** [comment]
@newnottakenname this https://github.com/driftyco/ionic/pull/6686 may help you once it is merged. stay tuned.

**Comment by newnottakenname** [comment]
@jgw96 added it to the issue
@manucorporat Even if it doesn't, it looks really cool!

**Comment by manucorporat** [comment]
@newnottakenname so, since (swipe) is a hammer event provided by angular 2 directly, we can't do much, but looks like hammer defaults can be configured in angular 2. 

Check out this:
https://github.com/angular/angular/pull/7924
and this:
https://github.com/hammerjs/hammer.js/wiki/Tips-&-Tricks#improve-dragswipetransform-experience

also, https://github.com/driftyco/ionic/pull/6686 was merged, so you will be able to subscribe to the new  (ionSlide) and (ionSwipe) events in beta9, these methods should work perfectly with scrolling.

I will close this as Won't fix.
